gradle/versions.gradle

```diff
@@ -13,7 +13,7 @@
 version = '0.17.0-SNAPSHOT'
 
 ext.versions = [
-	'xtend_lib': '2.24.0',
+	'xtend_lib': '2.28.0',
 	'guava': '[14.0,31)',

```

Comment:
is this one still used? should we update it?

Reply:
yes it seems to be used in the manifests. should we also bump minimal version here?

Reply:
What would you want the minimum to be? Perhaps `30`?

Reply:
the version ranges in xtend/xbase.lib are `[30.1.0,31.0.0)` thus i assume something like `[30.1,31)` but i am not sure how nice ranges would look like @jonahgraham do you have any preferences?

Reply:
I've updated the guava range to `[30.1,31)`.

Reply:
I have no preference.